core/domain/feedback_services_test.py

```diff
@@ -344,6 +344,27 @@ def test_get_thread_summaries(self):
         # Check if the number of unread messages is equal to 1.
         self.assertEqual(number_of_unread_threads, 1)
 
+    def test_get_thread_summaries_increments_message_count(self):
+        thread_id_1 = feedback_services.create_thread(
+            'exploration', self.EXP_ID_1, None,
+            self.EXPECTED_THREAD_DICT['subject'], 'not used here')
+        thread = feedback_models.GeneralFeedbackThreadModel.get(thread_id_1)
+        thread.message_count = 0
+        thread.put()
+        thread_id_2 = feedback_services.create_thread(
+            'exploration', self.EXP_ID_2, None,
+            self.EXPECTED_THREAD_DICT['subject'], 'not used here')
+        thread = feedback_models.GeneralFeedbackThreadModel.get(thread_id_2)
+        thread.message_count = 0
```

Comment:
Why do this?

Reply:
This is to prove that message_count gets updated on calling get_thread_summaries()